--- common.cc	2021-04-23 10:54:11.000000000 +0800
+++ common.patch.cc	2021-04-23 11:03:01.000000000 +0800
@@ -38,17 +38,17 @@
 #include <stdio.h>
 #include <vector>
 
-#ifdef _WIN32
-#ifndef WIN32_LEAN_AND_MEAN
-#define WIN32_LEAN_AND_MEAN  // We only need minimal includes
-#endif
-#include <windows.h>
-#define snprintf _snprintf    // see comment in strutil.cc
-#elif defined(HAVE_PTHREAD)
+// #ifdef _WIN32
+// #ifndef WIN32_LEAN_AND_MEAN
+// #define WIN32_LEAN_AND_MEAN  // We only need minimal includes
+// #endif
+// #include <windows.h>
+// #define snprintf _snprintf    // see comment in strutil.cc
+// #elif defined(HAVE_PTHREAD)
 #include <pthread.h>
-#else
-#error "No suitable threading library available."
-#endif
+// #else
+// #error "No suitable threading library available."
+// #endif
 #if defined(__ANDROID__)
 #include <android/log.h>
 #endif
